dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/mock/MockUserService.java

```diff
@@ -179,6 +179,21 @@ public UserCredentials getUserCredentialsByUsername( String username )
         return null;
     }
 
+    @Override
+    public UserCredentials getUserCredentialsWithEagerFetchAuthorities( String username )
+    {
```

Comment:
Could you just reuse `getUserCredentialsByUsername` method here and then call getAllAuthorities ?